examples/toolchains/rust/patches/001-rules-rust-crate-universe.patch

```diff
@@ -24,36 +24,33 @@ index 28f6d14..a2eeb3a 100644
 +        rust_toolchain_rustc_template = rust_toolchain_rustc_template,
      )
 diff --git crate_universe/repositories.bzl crate_universe/repositories.bzl
-index 0dded9b..fefc00a 100644
+index c06e06d9..12863402 100644
 --- crate_universe/repositories.bzl
 +++ crate_universe/repositories.bzl
-@@ -31,16 +31,27 @@ _MANIFESTS = [
-     "@rules_rust//crate_universe/tools/urls_generator:Cargo.toml",
- ]
+@@ -7,7 +7,12 @@ load("//crate_universe/3rdparty/crates:crates.bzl", _vendor_crate_repositories =
+ load("//crate_universe/private:vendor_utils.bzl", "crates_vendor_deps")
+ load("//crate_universe/tools/cross_installer:cross_installer_deps.bzl", "cross_installer_deps")
  
 -def crate_universe_dependencies(rust_version = rust_common.default_version, bootstrap = False):
 +def crate_universe_dependencies(
-+    rust_version = rust_common.default_version,
-+    bootstrap = False,
-+    rust_toolchain_cargo_template = "@rust_{system}_{arch}//:bin/{tool}",
-+    rust_toolchain_rustc_template = "@rust_{system}_{arch}//:bin/{tool}",
-+    ):
++        rust_version = rust_common.default_version,
++        bootstrap = False,
++        rust_toolchain_cargo_template = "@rust_{system}_{arch}//:bin/{tool}",
++        rust_toolchain_rustc_template = "@rust_{system}_{arch}//:bin/{tool}",
++):
      """Define dependencies of the `cargo-bazel` Rust target
  
      Args:
-         rust_version (str, optional): The version of rust to use when generating dependencies.
-         bootstrap (bool, optional): If true, a `cargo_bootstrap_repository` target will be generated.
-+        rust_toolchain_cargo_template (str, optional): The template to use for finding the host `cargo` binary.
-+        rust_toolchain_rustc_template (str, optional): The template to use for finding the host `rustc` binary.
```

Comment:
why remove those from the patch?

Reply:
This was not intentional. The idea was to get it working, this patch being for internal use, not to be included upstream. I'm going to restore this and add the lines in another place where the API is also extended (`cargo_bazel_bootstrap`, which did not have such lines in the old patch).